src/app/components/LinkData/index.jsx

```diff
@@ -0,0 +1,80 @@
+import React, { Fragment } from 'react';
+import { string, number } from 'prop-types';
+
+const LinkData = ({
+  type,
+  seoHeadline,
+  firstPublished,
+  lastUpdated,
+  optimoUrn,
+  service,
+}) => {
+  const imgObject = 'ImageObject';
+  const newsMediaOrg = 'NewsMediaOrganization';
+  const url =
+    'https://www.bbc.com/news/special/2015/newsspec_10857/bbc_news_logo.png?cb=1';
+
+  const optimoId = optimoUrn.split(':').pop();
```

Comment:
To remove duplication of this logic between the Metadata Container and here, you could use LinkData component in the container here: https://github.com/bbc/simorgh/blob/latest/src/app/containers/Metadata/index.jsx#L69
So passing the values `optimoId`, `timeFirstPublished`, `timeLastPublished` into this component.

Reply:
great call, thanks :D I made the changes, but the <script> tag still doesn't show in the head.. I tried copying exactly how the `Metadata` component uses Helmet (ie. with the htmlAttributes) but it didn't work. 

https://github.com/bbc/simorgh/blob/a6a08046c37b60d231c981c97e1a99407f32162c/src/server/Document/component.jsx#L14 this is coming through as an empty array... not sure why 😬 and not really sure how to fix it...